### PR TITLE
Feature/stale headers

### PR DESCRIPTION
**Pull request by tomasinchoo**

### CODE REVIEW

File: fastly.services.yml [hunk 1]
```diff
@@ -16,7 +16,7 @@ services:
     tags:
       - { name: event_subscriber }
   fastly.cache_tags.add_soft_purge_headers:
```
Comment (greylabel) [inline fastly.services.yml]: Would you want to rename the service to ` fastly.cache_tags.add_stale_headers` and update anywhere that called the old service name, just for consistency?

File: config/schema/fastly.schema.yml [hunk 2]
```diff
@@ -11,6 +11,9 @@ fastly.settings:
     purge_method:
       type: string
       label: 'Fastly Purge Method'
+    stale_while_revalidate:
```
Comment (greylabel) [inline config/schema/fastly.schema.yml:14]: Should we assume this should be set to the value that may have been previously set for `stale_if_error`? Could write an update hook, if so.

**Review by greylabel — COMMENTED** [review]
See comments, but LGTM overall.

File: config/schema/fastly.schema.yml (hunk 2 above)
Comment (vvuksan) [reply config/schema/fastly.schema.yml:14]: This is just the button to enable/disable.

**Pull request merged**